Prose/ProseLib/Sources/SharedModels/User.swift

```diff
@@ -0,0 +1,62 @@
+//
```

Comment:
Why not moving it in `ProseCore`? This is where it'll live in the future. I had put it in `ProseCoreStub` to represent the fact that it will come from `ProseCore`.

Reply:
Done.